.github/workflows/lint-test.yaml

```diff
@@ -50,11 +50,32 @@ jobs:
       - name: Run chart-testing (install)
         run: |
           ct install --config ct.yaml \
-            --helm-extra-set-args "
+              --helm-extra-set-args "
```

Comment:
Are the quites in the back intentional? Also should you include end of line, i.e. `\`?